internal/entity/dto/v1/domain_test.go

```diff
@@ -0,0 +1,77 @@
+package dto
+
+import (
+	"testing"
+
+	"github.com/go-playground/validator/v10"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateAlphaNumHyphenUnderscore(t *testing.T) {
```

Comment:
what is the expectation for profile name starting with _ (underscore) or - ( hyphen) ex:  _myprofile,  -myprofile . Are they allowed, have test-cases around them ?